Detections/W3CIISLog/MaliciousAlertLinkedWebRequests.yaml

```diff
@@ -0,0 +1,64 @@
+id: fbfbf530-506b-49a4-81ad-4030885a195c
+name: Malicious web application requests linked with MDATP alerts
+description: |
+  'Takes MDATP alerts where web scripts are present in the evidence and correlates with requests made to those scripts
+  in the WCSIISLog to surface new alerts for potentially malicious web request activity.
+  The lookback for alerts is set to 1h and the lookback for W3CIISLogs is set to 7d. A sample set of popular web script extensions
+  has been provided in scriptExtensions that should be tailored to your environment.'
+severity: Medium
+requiredDataConnectors:
+  - connectorId: MicrosoftDefenderAdvancedThreatProtection
+    dataTypes:
+      - SecurityAlert
+  - connectorId: AzureMonitor(IIS)
+    dataTypes:
+      - W3CIISLog      
+queryFrequency: 1h
+queryPeriod: 7d
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - Persistence
+relevantTechniques:
+  - T1100
+query: |
+
+  let alertTimeWindow = 1h;
+  let logTimeWindow = 7d;
+  // Define script extensions that suit your web application environment - a sample are provided below
+  let scriptExtensions = dynamic([".php", ".jsp", ".js", ".aspx", ".asmx", ".asax", ".cfm", ".shtml"]); 
+  let alertData = SecurityAlert 
```

Comment:
Only thought is that because this is re-used a couple times later you could wrap this in materalize(), this may help with future perf/compute, but not a requirement at this time.